rdr/cnv-workload/vm-resources/vm-workload-1/vm-1-pvc.yaml

```diff
@@ -13,5 +13,5 @@ spec:
   resources:
     requests:
       storage: 20Gi
-  storageClassName: ocs-storagecluster-ceph-rbd-virtualization
+  storageClassName: rbd-cnv-custom-sc-r2
```

Comment:
you are changing SC in wrong folder.

Reply:
thanks for noticing, it was by mistake..fixed it